install.yaml

```diff
@@ -219,18 +222,17 @@ post_install_actions:
   {{ $phpversion := trimPrefix "php:" .platformapp.type }}
   {{ if .platformapp.runtime.extensions }}
   {{ $phpextensions := without .platformapp.runtime.extensions "blackfire" "pdo_pgsql" "sodium" }}
-  {{range $extension := $phpextensions }}
+    {{range $extension := $phpextensions }}
```

Comment:
Are the WS intended?

Reply:
No, not intended. No harm done, but not intended.